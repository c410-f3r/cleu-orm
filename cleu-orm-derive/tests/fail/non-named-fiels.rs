
#[derive(cleu_orm_derive::TableParams)]
pub struct Foo(i32);

fn main() {
}