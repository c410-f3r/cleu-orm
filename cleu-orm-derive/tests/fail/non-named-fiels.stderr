error: Table element must be a structure with named fields
 --> tests/fail/non-named-fiels.rs:3:1
  |
3 | pub struct Foo(i32);
  | ^^^^^^^^^^^^^^^^^^^^
